Removing created-by label deprecated since 1.9

yilmi opened the pull request:
**Why this change?**
This changes removes the `created-by` annotation given that it has been deprecated since 1.9, see [Kubernetes PR# 54445](https://github.com/kubernetes/kubernetes/pull/54445).

This deprecation was announced in the following release notes:
- [Change log Kubernetes 1.8 - Deprecations](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.8.md#deprecations)
- [Change log Kubernetes 1.8 - Other notable changes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.9.md#other-notable-changes-14)

Sea-n commented:
Hey @yilmi,

You have to sign the CLA so that we can review this PR and move forward.
Just click on the red NOT COVERED button in the first comment.

After signing the CLA, you may use this command to refresh the status:
/easycla

If you have any problem, please take a look at [the CLA tutorial](https://git.k8s.io/community/CLA.md).

Arhell reviewed (commented):
I also recommend breaking it down for each language.

yilmi commented:
@Sea-n I'm trying to get a corporate CLA signed on my end, but may switch back to submitting as an individual contributor.

@Arhell sure, will do, thanks for the feedback

seokho-son commented:
Hi @yilmi 
Thank you for the contribution. :)

In case of Korean contents, Korean localization team is handling outdated Korean contents. (we have a routine to update all Korean contents) So, the changes in English document will be synced to Korean. So, could you let us handle the changes in contents/ko/* ?

bradtopol commented:
Hi @yilmi Thank you for submitting the PR. Please sign the CLA so we can accept it for review. Also, in general a PR should only target a single language.  So this PR needs to be broken up into one pr per language (ko, zh, english)
/assign

yilmi commented:
Hi everyone, I moved to an individual contribution as the CLA signature for a corporate contributor can take some time.

@Sea-n / @bradtopol  - CLA for an individual contributor was just signed
@seokho-son - Thanks for the heads-up, I removed the changes to the Korean content :+1: 

Those changes would also have to be completed for the Chinese content, not sure how I could flag it for someone to pick it up.

Thanks!
Yassine

Sea-n commented:
Thanks for signing the CLA agreement and change just en files.
/remove-label zh ko

For Chinese pages, we have some other contributors working on localization and keep sync with upstream (English) version. No need to worry about it. 😊

Sea-n commented:
Oh, my bad
/remove-language zh ko

yilmi commented:
Hi @Arhell, @bradtopol let me know if there's something I should add. Thanks

sftim commented on content/en/docs/reference/labels-annotations-taints/_index.md:
```diff
@@ -35,6 +35,10 @@ The controller/user who created this resource.
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
 
+{{< note >}}
+Starting from v1.9, this annotation is deprecated.
```
```suggestion
Starting from v1.9, this label is deprecated.
```

sftim commented on content/en/docs/reference/labels-annotations-taints/_index.md:
```diff
@@ -35,6 +35,10 @@ The controller/user who created this resource.
 
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
```
```suggestion
```

sftim reviewed (commented):
Thanks.

Two suggestions.

sftim reviewed (commented):
Thanks

/lgtm
/approve
